wolfcrypt/src/tfm.c

```diff
@@ -1653,11 +1547,7 @@ int fp_submod(fp_int *a, fp_int *b, fp_int *c, fp_int *d)
    fp_int *t;
 #endif
 
-#ifdef WOLFSSL_SMALL_STACK
-   t = (fp_int*)XMALLOC(sizeof(fp_int), NULL, DYNAMIC_TYPE_BIGINT);
-   if (t == NULL)
-       return FP_MEM;
-#endif
+   WC_ALLOC_VAR_EX(t, fp_int, 1, NULL, DYNAMIC_TYPE_BIGINT, return FP_MEM);
```

Comment:
Should `WC_DECLARE_VAR` be used above?

Reply:
thanks for picking that up -- i was only transforming `#ifdefs` and not `#ifndefs`. fixed